src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -1396,3 +1412,16 @@ def saveCustomConfig(self):
         Save the config file based on current session values
         """
         config.save()
+
+    def resetProject(self):
+        """
+        Reset the project to an empty state
+        """
+        # file manager
+        self.filesWidget.reset()
```

Comment:
It might be safer to reset the perspectives before removing the data from the data explorer.

Reply:
Not really, since resetting the filesWidget includes sending signals to perspectives to deal with removed data, but I did it anyway, as it doesn't seem to affect the functionality.